### ISSUE TITLE
Hand Matching -- need to be able to match multiple records together (V 1.5 feature)

### ISSUE THREAD

**Issue by RDmitchell** [SEED-platform/seed#1385]
instance: staging
SHA: 
Org: LBNL 20

In V 1.5 it was possible, in the hand matching screen, to match an infinite number of records together.
![image](https://user-images.githubusercontent.com/6314950/26953579-09b42f64-4c60-11e7-8d16-157a9f6fc582.png)

In V 2.1, the program only allows matching one record to another record. I suppose it might be possible to make a "chain" of record that could be matched together, but from the user perspective, if they have multiple records that need to be matched together, they should be able to do the matching in one step (hand matching is painful enough as it is ).

In this case, I have already matched two records together in the "hand matching" screen
![image](https://user-images.githubusercontent.com/6314950/26953626-68437878-4c60-11e7-80dd-66d9a17858f2.png)

If I try to check a 2nd record (the one right below the record already checked), I get this message, saying that I can't merge the 2nd property without unmerging the first one. Users need to be able to match multiple records together.
![image](https://user-images.githubusercontent.com/6314950/26953654-9ca5df48-4c60-11e7-8c73-ddc75a76d0d7.png)

**Comment by RDmitchell** [comment]
This is related to #1313

**Comment by RDmitchell** [comment]
I am closing #1313, and adding the info from that issue to this one:
****
instance: dev1
SHA: 6f5e402
Org: LBNL 202

Folder with files and doc
https://drive.google.com/open?id=0B3fTKpZ9Dx7LcGxSRnk0VlloOUk

**Comment by RDmitchell** [comment]
@axelstudios -- you may have answered this before, but is there anyway to unmerged merged records? I can't remember.

**Comment by axelstudios** [comment]
@RDmitchell Yes, when you go to the detail page for a merged record you'll find a new `Unmerge Last` button:
![unmerge last](https://user-images.githubusercontent.com/411466/35045382-e5746d94-fb50-11e7-99bb-47b114003767.png)

We don't allow unmerging from the middle of long merge trees anymore to reduce complexity, but you can repeatedly unmerge the last-merged record

**Comment by RDmitchell** [comment]
Right, the big orange button! How could I miss it? Thx!!

**Comment by RDmitchell** [comment]
Instance: dev1
SHA: d266f3a
Org: ID = 10, Montgomery County
Cycle: 2016 Compliance Year

The new matching works SO MUCH BETTER than the way it used to work. Thanks @axelstudios !!!

I have tested merging through the inventory list (including changing the merge order), unmerging through the inventory detail, unpairing through the inventory detail.

I have raised a few issues which @axelstudios is looking at, but basically this is working really well. I am closing this issue and we can open new ones for refinements and/or bugs. 

Thanks again @axelstudios -- a major improvement in the program !
